java/src/main/java/com/example/perf/SimpleHttp.java

```diff
@@ -0,0 +1,86 @@
+package com.example.perf;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.nio.ByteBuffer;
+import java.nio.CharBuffer;
+import java.nio.channels.SelectionKey;
+import java.nio.channels.Selector;
+import java.nio.channels.ServerSocketChannel;
+import java.nio.channels.SocketChannel;
+import java.nio.charset.Charset;
+import java.nio.charset.CharsetEncoder;
+import java.text.SimpleDateFormat;
+import java.util.TimeZone;
+import java.util.Calendar;
+import java.util.Iterator;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class SimpleHttp {
+
+    static final String headerTemplate =
+        "HTTP/1.1 200 OK\r\n" +
+        "Date: %s\r\n" +
+        "Server: Java NIO\r\n"+
+        "Connection: close\r\n"+
+        "Content-Length: %d\r\n" +
+        "\r\n";
+    static final String contentTemplate = "Speed Test Response\n"+
+            "This is request #%06d\n"+
+            "Current time is %s";
+
+    static final SimpleDateFormat RFC3339 =
+//            new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
+                new SimpleDateFormat("EEE MMM dd HH:mm:ss z yyyy");
```

Comment:
Do not change the datetime format. The response value should be the same between different tests